mongo/integration/mtest/setup.go

```diff
@@ -300,19 +311,50 @@ func addCompressors(uri string) string {
 	return addOptions(uri, "compressors=", comp)
 }
 
+func addAuthCredentials(uri string) (string, error) {
```

Comment:
I suggest renaming this to clarify this only applies to serverless auth.

```suggestion
func addServerlessAuthCredentials(uri string) (string, error) {
```